controller/notifications/index.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright © 2024, F2FINTECH. ALL RIGHTS RESERVED.
+ *
+ * This software is the confidential information of F2FINTECH., and is licensed as
+ * restricted rights software. The use, reproduction, or disclosure of this software is subject to
+ * restrictions set forth in your license agreement with F2 FINTECH.
+ */
+const NotificationModel = require("../../model/notifications");
```

Comment:
spelling should be notification, always use singular form as used in all other folders,